plugins/webp-uploads/tests/test-uninstall.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Tests for webp-uploads plugin uninstall.php.
+ *
+ * @package webp-uploads
+ */
+
+class Test_WebP_Uploads_Uninstall extends WP_UnitTestCase {
+
```

Comment:
The culprit appears to be in the uninstall test for single site - the `WP_UNINSTALL_PLUGIN` constant isn't defined, which causes `uninstall.php` to exit early:
https://github.com/WordPress/performance/blob/04f8b04510fce812ab34b5b3b5f0c9128ed4c727/plugins/webp-uploads/uninstall.php#L9-L12

This premature exit terminates the entire PHP runtime execution, preventing the coverage report from being generated. Interestingly, the test itself doesn't fail, which can be confusing at first.

A possible fix would be to define the constant in the test class like this:
```suggestion

	/**
	 * Runs the routine before setting up all tests.
	 */
	public static function set_up_before_class(): void {
		parent::set_up_before_class();
	
		// Mock uninstall const.
		if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
			define( 'WP_UNINSTALL_PLUGIN', 'Yes' );
		}
	}

```